includes/Renderer/Story/HTML.php

```diff
@@ -138,25 +138,6 @@ public function add_web_stories_amp_content_sanitizers( $sanitizers ) {
 		return $sanitizers;
 	}
 
-	/**
```

Comment:
Just some dead code I found while working on this. This method is nowhere used. Leftover from when this code was moved to the `\Google\Web_Stories\Integrations\AMP` class.